test/arrays/reindex/emptyReindex.good

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is B's domain supposed to be ``{10..9}`` ? If so, can you print it out here as well?